README.md

```diff
@@ -605,8 +638,8 @@ The default layouts are [all of the kinds](leftwm-core/src/layouts/mod.rs#L21) d
 | Issue                              | Description                                                                                                              |                        Solution                        |
 | ---------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------------: |
 | LeftWM not listed by login manager | You likely need to add the xsessions file to the right folder.                                                           | See [installation](#installation-with-package-manager) |
-| No config.toml file exists         | LeftWM does not always ship with a `config.toml`. You will need to execute LeftWM at least once for one to be generated. |           Try the following: `leftwm-worker`           |
-| Config.toml is not being parsed    | LeftWM ships with a binary called leftwm-check. It might not be installed by the AUR.                                    |           Try the following: `leftwm-check`            |
+| No `config.ron` file exists         | LeftWM does not always ship with a `config.ron`. You will need to execute LeftWM at least once for one to be generated. |           Try the following: `leftwm-worker`           |
+| `config.ron` is not being parsed    | LeftWM ships with a binary called leftwm-check. It might not be installed by the AUR.                                    |           Try the following: `leftwm-check`            |
```

Comment:
Column separators are now misaligned.

Reply:
Ah my mistake, will fix later (hopefully this evening)